calico_kubernetes/calico_kubernetes.py

```diff
@@ -356,9 +369,16 @@ def _get_container_info(self, container_name):
     def _get_container_pid(self, container_name):
         return self._get_container_info(container_name)["State"]["Pid"]
 
+    def _get_ipset_from_endpoint(self, endpoint):
```

Comment:
What was the reasoning for combining this into a helper method? It looks like there is only one place using this. 


I'd leave this as it was.